pkgs/games/grapejuice/default.nix

```diff
@@ -87,9 +116,10 @@ python3Packages.buildPythonApplication rec  {
 
   meta = with lib; {
     homepage = "https://gitlab.com/brinkervii/grapejuice";
-    description = "A wine+Roblox management application";
+    description = "Simple Wine+Roblox management tool";
     license = licenses.gpl3Plus;
     platforms = platforms.linux;
-    maintainers = with maintainers; [ artturin ];
+    maintainers = with maintainers; [ artturin helium ];
   };
 }
+
```

Comment:
```suggestion
```

i recommend setting up your editor so that it automatically removes trailing newlines and whitespaces

Reply:
I use https://editorconfig.org/ with vim. Not sure how it missed that. I'll fix it :+1:

Reply:
i use that too + https://github.com/mcauley-penney/tidy.nvim

Reply:
thanks,